qemu/tests/timerdevice_boot.py

```diff
@@ -42,6 +44,16 @@ def verify_guest_clock_source(session, expected):
             raise error.TestFail(
                 "Guest didn't use '%s' clocksource" % expected)
 
+    def verify_base_date_system_time(time_command, time_filter, time_format):
+        (host_time, guest_time) = utils_test.get_time(session, time_command,
+                                                      time_filter_re, time_format)
+        guest_time = time.strftime('%Y-%m-%d %H', time.localtime(guest_time))
+        os_type = params["os_type"]
+        if os_type == "linux" and guest_time != "2006-06-17 08":
```

Comment:
Thanks @luckyh 
Update the code and use "date -u" to get the utc time in the guest.